src/com/company/Main.java

```diff
@@ -33,43 +55,34 @@ public static String readFileAsString(String fileName)
               } catch (IOException e) {
                   e.printStackTrace();
               }
-              return data;
+              return data.replaceAll("[^a-zA-Z0-9@]", " ");
+
     }
 
-        public static void checkforsupport(File file)
+        public static boolean isFileFormatSupported(File file)
         {
-
-            if (file.getName().endsWith(".txt")) {
-                System.out.println("file supported");
-
-            }
-
-            else
-                {
-                    System.out.println("file not supported");
-                }
+            return (file.getName().endsWith(".txt"));
         }
 
-        public static void checkforexistance(File file) {
-            if (file.exists()) {
-
-                System.out.println("file exists");
-            }
-            else {
-                System.out.println("file doesn't found ");
-            }
-        }
-        public static void avoidspecialchar(String data) {
-            data = data.replaceAll("[^a-zA-Z0-9@]", " ");
+        public static boolean isFileExists(File file) {
+           return file.exists();
```

Comment:
Why have a different method if it is just a one line check? You can include file.exists() directly in the if condition

Reply:
we can do that and I have done that in recently pushed code.